src/System.Runtime/tests/System/DateTimeTests.cs

```diff
@@ -1229,7 +1229,21 @@ public static void InvalidDateTimeStyles()
             AssertExtensions.Throws<ArgumentException>("style", () => DateTime.ParseExact(strDateTime, formats, provider, style));
         }
 
-        public static IEnumerable<object[]> Parse_ValidInput_Suceeds_MemberData()
+        [Fact]
+        public static void EnsureTryParseNotThrowing()
```

Comment:
Nit: it'd be nice to have a more specific name here, about it not throwing when at a boundary and factoring in the timespan would cause it to overflow/underflow.

Reply:
Would the name TestTryParseAtBoundaries would reflect what you are suggesting? I can add extra Assert to ensure it works on the valid boundary?